package.json

```diff
@@ -53,102 +53,89 @@
     "serve:angular": "nx serve angular",
     "prepack": "yarn build:core && yarn build:all",
     "test": "nx run-many --target=test --all",
-    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main && yarn build:core",
-    "ng": "nx"
+    "postinstall": "ngcc --properties es2020 browser module main"
```

Comment:
We probably need to revert back this line since it might be important for NX I am not sure what it does but Rares has added it:

`"postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main && yarn build:core"`

Reply:
Maybe it's okay that we don't have it now? I guess if the Angular example works, we're all good? They talk about it in the docs as a way to migrate from an existing Angular structured project if I'm not mistaken: https://nx.dev/migration/migration-angular#using-the-nx-cli-while-preserving-the-existing-structure